imports/server/publications/pendingGuessesForSelf.ts

```diff
@@ -83,23 +84,14 @@ class PendingGuessWatcher {
   }
 }
 
-// Publish pending guesses enriched with puzzle and hunt. This is a dedicated
```

Comment:
This comment seems like helpful context for understanding the publication; is there a reason you're removing it?

Reply:
Huh, no it must have gotten dropped by accident. I'll add it back, although my inclination is to put high-level commentary about how to interact with a publication on the declaration, rather than the definition.